app/src/main/java/com/example/book/data/repository/source/remote/fetchJson/ParseDataWithJson.kt

```diff
@@ -0,0 +1,59 @@
+package com.example.book.data.repository.source.remote.fetchJson
+
+import android.util.Log
+import com.example.book.data.model.Book
+import org.json.JSONArray
+import org.json.JSONException
+import org.json.JSONObject
+
+class ParseDataWithJson {
+
+    fun parseJsonToListData(jsonObject: JSONObject?, keyEntity: String): Any {
+        val data = mutableListOf<Book>()
```

Comment:
Không được để kiểu trả về là any, như vậy sẽ không đảm bảo type safety. Dùng Generic type đi em
```suggestion
    fun <T> parseJsonToListData(jsonObject: JSONObject?, keyEntity: String): T {
        val data = mutableListOf<Book>()
```